data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
-
 set -ex
 
-# We should switch to openshift builds of these images when ready ref
-# https://github.com/openshift/installer/issues/2090
-IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
-IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
-IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
-COREOS_DOWNLOADER_IMAGE=${COREOS_DOWNLOADER_IMAGE:-"quay.io/openshift-metal3/rhcos-downloader:master"}
+. /usr/local/bin/release-image.sh
```

Comment:
cc @wking for bash knowledge..
 How is this different from `source`

Reply:
They are identical in bash, AFAIK. I'm happy to switch to `source` if there's a preference. https://ss64.com/bash/source.html indicates it's more portable, but since we're specifying /bin/bash at the top I'm sure it';s fine.

Reply:
+1 to @stbenjam's comment.  Here's [`.` in POSIX][1], and [here][2] is them saying `source` is unspecified.  But since we're using Bash, we don't have to be POSIX-compliant (although personally I prefer POSIX-compliance in places like this where it's no hardship).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#dot
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_01_01